### PR TITLE
Navigation: View Experiences

### PR DESCRIPTION
**Pull request by anoura12**

### CODE REVIEW

**Review by jhlink — CHANGES_REQUESTED** [review]
Hey @anoura12 ! 

Strong work!

Unfortunately, when I ran your PR, I got this NoReverseMatch error. 

CleanShot 2021-07-29 at 06 38 21@2x


It seems that all the references to `public_experiences` in the templates haven't been changed to `view_experiences`. Please re-request a review for me when this is finished and the docker instance doesn't show an error on start. :)

**Comment by anoura12** [comment]
@jhlink I have made the changes in all files. Could you take a look at them?

**Comment by anoura12** [comment]
@jhlink I think I've resolved the merge conflicts. Is it good to merge now?

**Comment by jhlink** [comment]
@anoura12 I'll take a look later today!

**Comment by jhlink** [comment]
PR is good. Merging.

**Review by jhlink — COMMENTED** [review]
@anoura12 , please create another PR with the changes I've specified here. 
Unfortunately, I had to revert the PR because you've overwritten Mridu's PR.

File: server/apps/main/templates/main/moderate_public_experiences.html [hunk 1]
```diff
@@ -54,11 +54,15 @@
 	<nav class="navbar">
 		<a class="navbar-brand" href="#" </a>
              <a style="color:#000000;"href="{% url 'main:index' %}">Home</a>
-             <a style="color:#000000;"href="{% url 'main:public_experiences'%}">View Experiences</a>
-             <a style="color:#000000;"href="{% url 'main:moderate_public_experiences' %}" class="active">Moderate Experiences</a>
-             <a style="color:#000000;"href="{% url 'main:upload' %}" class="active">Share Experiences</a>
-			 <a style="color:#000000;"href="{% url 'main:my_stories' %}" class="active">My Stories</a>
-             <a style="color:#000000;"href="{% url 'main:logout' %}">Logout</a>
+             <a style="color:#000000;"href="#" class="active">About</a>
```
[the remark this answers is not in the document]
Comment (anoura12) [reply server/apps/main/templates/main/moderate_public_experiences.html:57]: Yeah will do

**Pull request merged**